ci-operator/jobs/openshift/release/openshift-release-release-4.8-periodics.yaml

```diff
@@ -290,190 +290,72 @@ periodics:
   decoration_config:
     skip_cloning: true
   extra_refs:
-  - base_ref: release-4.8
+  - base_ref: master
```

Comment:
Once 4.8 branches, this should become `release-4.8` - I wonder whether we should just point `release-4.8` on the okd-machine-os to master for now, so this won't have to change here.

Reply:
There is no autosync mechanism, so until branch cut we'd have to do that manually for some time.

It can be done later, when 4.9 release is being prepared